admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/controller/DynamicFieldController.java

```diff
@@ -93,6 +93,22 @@ public ResponseWrapper<DynamicFieldConsolidateResponseDto> getDynamicFieldByName
 		responseWrapper.setResponse(dynamicFieldService.getDynamicFieldByNameAndLangcode(fieldName,langCode,withValue));
 		return responseWrapper;
 	}
+
+	@ResponseFilter
+	@GetMapping("/all/{fieldName}")
+	@ApiOperation(value = "Service to fetch all dynamic field value for all languages")
```

Comment:
Please correct the API description - This is a service to fetch one dynamic field in all the languages

Reply:
Done.